src/containers/Home/Home.js

```diff
@@ -64,15 +65,17 @@ const Home = ({ setCurrentProfile, user }) => {
           <div className="city-state">
             <div className="home-form-item">
               <div className="city-label">
-                <label>City</label>
-                <input
-                  id="city-input"
-                  type="text"
-                  name="city"
-                  value={city}
-                  required
-                  onChange={e => updateCity(e.target.value)}
-                />
+                <label>City
```

Comment:
I like this format, on the component I'm testing I'll be moving all my label/inputs to this structure